Source/Engine/Physics/Actors/RigidBody.cpp

```diff
@@ -327,12 +327,10 @@ void RigidBody::OnColliderChanged(Collider* c)
 
 void RigidBody::UpdateBounds()
 {
-    void* actor = _actor;
-    if (actor && PhysicsBackend::GetRigidActorShapesCount(actor) != 0)
-        PhysicsBackend::GetActorBounds(actor, _box);
```

Comment:
u know this is getting a BB from PhysicsBackend with is accurate where rb actor is 

new code is ignoring the PhysicsBackend bounding box